ai21/models/responses/thread_response.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import Optional, List, Literal
+
+from typing_extensions import TypedDict
+
+from ai21.models.ai21_base_model import AI21BaseModel
+
+
+MessageRole = Literal["assistant", "user"]
+
+
+class MessageContentText(TypedDict):
+    type: Literal["text"]
+    text: str
+
+
+class CreateMessagePayload(TypedDict):
```

Comment:
How about only `Message`?

Reply:
no :( 
when the user creates a message they can control only specific fields, while the `message` that returns from the API has some extra fields in it that aren't in the user's capability of changing.

Reply:
I was referring to the name of the object